lgsm/config-default/config-lgsm/mcserver/_default.cfg

```diff
@@ -152,8 +149,8 @@ glibc="null"
 ## Game Server Directories
 systemdir="${serverfiles}"
 executabledir="${serverfiles}"
-jarfile="${serverfiles}/minecraft_server.jar"
-executable="${jarfile}"
+prexecutable="java -Xmx${javaram}M -jar"
```

Comment:
The `prexecutable` is not in the eval function, and so the `javaram` is not used again.

Reply:
tested this and is working